<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>АлкоМетр</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="Lay_Btn">
      <item>
       <widget class="QPushButton" name="Btn_OpenBar">
        <property name="text">
         <string>Открыть Бар</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_OpenCoctailManager">
        <property name="text">
         <string>Открыть менеджер коктейлей</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Сделать коктейль&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Lay_AlcoItems">
      <item>
       <widget class="QComboBox" name="W_typeCoctail"/>
      </item>
      <item>
       <widget class="QComboBox" name="W_coctailName"/>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Chose">
        <property name="text">
         <string>Выбрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Lay_Table">
      <item>
       <widget class="QTableWidget" name="W_CoctailItems"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Lay_rightCoctail">
        <item>
         <widget class="QTextBrowser" name="W_About"/>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_CreateCoctail">
          <property name="text">
           <string>Сделать коктейль</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
